src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -63,6 +66,14 @@ export const PhoneNumberInput: FunctionComponent<{
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   onSubmit = () => {}
 }) => {
+  const phoneUtil = PhoneNumberUtil.getInstance();
```

Comment:
Just a nitpick, maybe this can be moved outside of the scope of the component since this is a singleton? We don't need to get a new instance every time this component is rendered.

Reply:
moved this to outside of scope so don't need to get new instance each time component is rendered.

Reply:
Thanks a lot!